lisa/search_space.py

```diff
@@ -452,6 +452,37 @@ def generate_min_capability_setspace_from_priority(
     return min_cap
 
 
+def check_boolean(
```

Comment:
The boolean is not like a searchable space, and there are several types about how to merge req/cap on boolean. This is just one of situation. It's better to implement it in DiskOptionSettings itself.